src/domain/ideas/templates/advancedBoards.tpl.php

```diff
@@ -290,370 +305,421 @@ function tilt_direction(item) {
-                                        <br />Last modified on <?php echo date_format(new DateTime($row["modified"]), "m/d/Y");?>
-                                        <?php if($row['milestoneHeadline'] != '') {?>
-                                            <br /> <hr />
-                                            <div class="row">
+                <div class="row" id="firstRow">
 
-                                                <div class="col-md-5" >
-                                                    <?php $this->e(substr($row['milestoneHeadline'], 0, 10)); ?> [...]
-                                                </div>
-                                                <div class="col-md-7" style="text-align:right">
-                                                    <?php echo $row['percentDone']; ?>% Complete
+                    <div class="col-md-2 reducePadding">
+                        <div class="column bgColumn">
+                            <h4 class="widgettitle title-primary">
+                                <?php if ($_SESSION['userdata']['role'] == 'admin' || $_SESSION['userdata']['role'] == 'manager') { ?>
+                                    <a href="/setting/editBoxLabel&module=idealabels&label=idea"
+                                       class="editLabelModal editHeadline"><i class="fas fa-edit"></i></a>
+                                <?php } ?>
+                                <?php $this->e($canvasLabels["idea"]); ?>
+                            </h4>
+                            <div class="contentInner status_idea">
+                                <?php foreach ($this->get('canvasItems') as $row) { ?>
+                                    <?php if ($row["box"] == "idea" || $row["box"] == "0") { ?>
+                                        <div class="ticketBox moveable" id="item_<?php echo $row["id"]; ?>">
+
+                                            <h4><a href="/ideas/ideaDialog/<?php echo $row["id"]; ?>" class="ideaModal"
+                                                   data="item_<?php echo $row["id"]; ?>"><?php $this->e($row["description"]); ?></a>
+                                            </h4>
+                                            <br/>
+                                            <?php echo nl2br($row["data"]); ?>
+                                            <br/><br/>
+
+
+                                            <span class="author"><span
+                                                        class="iconfa-user"></span> <?php $this->e($row["authorFirstname"]); ?> <?php $this->e($row["authorLastname"]); ?></span>&nbsp;
+                                            <span class="iconfa-comments"></span> <?php echo $row["commentCount"] ?> <?php echo $this->__("text.comments") ?>
+                                            <br/><?php echo $this->__("text.last_modified_on") ?> <?php echo date_format(new DateTime($row["modified"]), "m/d/Y"); ?>
```

Comment:
We'll need to internationalize the dateformat too. The first section in the ini file has the dateformats. You can replace m/d/Y with $this->__("language.dateformat")

Reply:
Ahh... Ok. Thank you.

Reply:
It should also use sprintf to allow for a different order of date + text: 
`<?=sprintf($this->__("text.modified_string"), date_format(...))?>`

and in the language file: 
text.modified_string = "Last Modified %1$s"

 %1$s specifies the first variable passed into sprintf for more than one just count up  %2$s, %3$s etc.